build.gradle.kts

```diff
@@ -1,6 +1,6 @@
 buildscript {
 	dependencies {
-		classpath("com.google.android.gms:oss-licenses-plugin:0.10.4")
+		classpath("com.google.android.gms:oss-licenses-plugin:0.10.6")
```

Comment:
Important change since this update. Licenses are listed only for `isDebuggable = false` builds 
![image](https://github.com/Android-Guild/Picover-KMP/assets/12697488/cc38e070-2427-4358-95ca-09fcc0dfb5cf)